hmpps-prisoner-search-indexer/helm_deploy/values-preprod.yaml

```diff
@@ -30,3 +30,6 @@ generic-prometheus-alerts:
 cron:
   snapshot_namespace: hmpps-prisoner-search-prod
   suspend_restore: false
+  index_refresh_schedule: "05 17 * * 1-5"
```

Comment:
don't need jobs running at the weekend - especially since every other Sunday night pre-prod will be out of sync since NOMIS database has been refreshed from production